### PR TITLE
add blocks and callback histogram metrics in ocr2vrf coordinator

### PR DESCRIPTION
**Pull request by jinhoonbang**
add blocks and callback histogram metrics in ocr2vrf coordinator

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go [hunk 1]
```diff
@@ -35,6 +37,41 @@ var (
...
+		64,
+		128,
+		256,
+		512,
+		1024,
+		2048,
+	}
+	promBlocksToReport = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "ocr2vrf_coordinator_blocks_to_report",
+		Help:    "Number of unfulfilled blocks in ReportBlocks",
```
Comment (vreff) [inline core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:57]: nit: it's not just the "unfulfilled blocks" or callbacks, it's the unfulfilled or inflight blocks that fit in the current batch.

Comment (jinhoonbang) [reply core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go:57]: true 👍

**Review by vreff — APPROVED** [review]

**Review by makramkd — COMMENTED** [review]
Should we add prom metrics for timings of each method in the coordinator? We log the duration of each method right now, but might be more useful to time them in prom for easier analysis.

**Comment by jinhoonbang** [comment]
> Should we add prom metrics for timings of each method in the coordinator? We log the duration of each method right now, but might be more useful to time them in prom for easier analysis.

yeah agreed. Will add those !

**Review by makramkd — APPROVED** [review]

**Pull request merged**